src/preamble.js

```diff
@@ -593,17 +593,16 @@ function abort(what) {
   }
 #endif
 
-  what += '';
+  what = 'Aborted(' + what + ')';
```

Comment:
```suggestion
  what = 'abort(' + what + ')';
```
This is the syntax of the actual call, which I think might help some people understand what happened. It's also shorter.

Reply:
I was taking a leaf out of linux's book:

```
$ clang ./test.c
$ ./a.out
Aborted
```

I kind of like it because its easier to grep for (i.e. I can easily find all the tests where the output is being checked without matching all the calls to abort), but that probably isn't very relevant.

Reply:
Would you be OK with leaving this?

Looks like I could easily change it with `s/Aborted/abort(/`.... (but note that changing it back would not be so easy because of all the other places `abort(` occurs :)

Reply:
I don't feel strongly about the text, sgtm.